### DISCUSSION TITLE
Add animation to double click window title to maximize?

### DISCUSSION THREAD

**Discussion by yunchong-song** [rxhanson/Rectangle#1216] [Ideas]
Hi, this is an awesome app, it it possible to add animation to double click window title to maximize? just like in the drag to resize function.

**Comment by rxhanson** [comment]
Thanks for the feedback. It's possible, but it would be a little strange in that it would involve first animating moving the window to the correct corner, then animating resizing the window. It's not really something that I have plans to add in, but I would be fine with merging a good pull request with this functionality as a terminal-command configurable setting.
